core/interfaces/i_rendered_element.ts

```diff
@@ -4,18 +4,15 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/** @internal */
 export interface IRenderedElement {
   /**
-   * @returns The root SVG element of htis rendered element.
+   * @returns The root SVG element of this rendered element.
```

Comment:
Add a note to the top-level PR description that this PR also makes `IRenderedElement` public.

Reply:
Done